packages/gatsby-cli/cli.js

```diff
@@ -1,3 +1,34 @@
 #!/usr/bin/env node
 
```

Comment:
Can we add a comment here that code in this file should not use any of "new" node/js features so it executes correctly regardless of used node version (well I guess up to a reasonable point) and doesn't crash unexpectedly? (that would be pretty meta to crash on version checking code because of wrong/old versions)